Source/AliveLib/Io.cpp

```diff
@@ -26,10 +29,29 @@ IO_FileHandleType IO_Open(const char* fileName, const char * mode)
         fileName += 2;
     }
 
+	std::string fixedPath(fileName);
+
+	// lowercase our string.
+	std::transform(fixedPath.begin(), fixedPath.end(), fixedPath.begin(),
```

Comment:
if this is unconditional its probably gonna break mac/linux

Reply:
batch file to lowercase all the file names? thats what ive done. unless theres a way to open files case insenstive.

Reply:
There is - but even still its gonna cause a regression if its merged as-is. Old alive proj has code to do a dir listing and then pick a file that matches any case.